gtp5g.c

```diff
@@ -812,6 +834,15 @@ static int pdr_fill(struct gtp5g_pdr *pdr, struct gtp5g_dev *gtp, struct genl_in
         pdr->seid = 0;
     }
 
+    /* If information has GTP5G_PDR_URR_ID, it means that user use the API which support 
```

Comment:
/*
     * For backward compatability: 
     * If information has GTP5G_PDR_URR_ID,
     * it means that user use the API which support 
     * URR and BAR feature. Otherwise, use the older API.
     */